Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.cpp

```diff
@@ -515,7 +516,31 @@ namespace Multiplayer
          || GetAgentType() == MultiplayerAgentType::DedicatedServer)
         {
             // We use a temporary userId over the clients address so we can maintain client lookups even in the event of wifi handoff
-            NetworkEntityHandle controlledEntity = SpawnDefaultPlayerPrefab(packet.GetTemporaryUserId());
+            IMultiplayerSpawner* spawner = AZ::Interface<IMultiplayerSpawner>::Get();
```

Comment:
We should make the filtered entity system and this system agree on how they register with the multiplayer gem just for API consistency. I don't really have a preference one way or another which style we end up picking, but try and work out with Olex which style you folks would like to move forward with.

Reply:
I'll take a look through and come up with some options.

Reply:
AZ::Interface::Get works as well, I assume that it won't affect perf versus a raw pointer but if it does, then we would need to know how much.